### PR TITLE
Added a plan drawing to the README

### PR DESCRIPTION
**Pull request by ifireball**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,116 @@
...
+
+1. Read the *UserSignup* resources from the host cluster (Via a K8s API or CLI
+   call) and generate a table mapping from a cluster username (As found in the
+     `status.compliantUsername` field) to SSO user ID (As could be found in the
+       `toolchain.dev.openshift.com/sso-user-id` annotation).
+2. Upload that table to a Splunk KV store (Via the REST API) so it can be used
+   via the Splunk `lookup` command.
+3. Run a Splunk query to extract all the interesting user activity events from
+   the API server audit logs while also converting the cluster usernames to SSO
+   user IDs (More details about the needed query blow).
```
Comment (Omeramsc) [inline README.md:72]: s/blow/below

File: README.md [hunk 2]
```diff
@@ -1,2 +1,116 @@
...
+   resources
+
+We can send details about the users' activity as seen via the cluster API
+server by doing the following on a periodic basis:
+
+1. Read the *UserSignup* resources from the host cluster (Via a K8s API or CLI
+   call) and generate a table mapping from a cluster username (As found in the
+     `status.compliantUsername` field) to SSO user ID (As could be found in the
+       `toolchain.dev.openshift.com/sso-user-id` annotation).
+2. Upload that table to a Splunk KV store (Via the REST API) so it can be used
```
Comment (Omeramsc) [inline README.md:68]: s/Via/via

**Review by Omeramsc — APPROVED** [review]
LGTM, comments contain only typos.

**Review by eisraeli — APPROVED** [review]
Except the minor typo the needs to be fixed, LGTM

File: README.md [hunk 3]
```diff
@@ -1,2 +1,116 @@
...
+   via the Splunk `lookup` command.
+3. Run a Splunk query to extract all the interesting user activity events from
```
Comment (Omeramsc) [inline README.md:70]: This PR might not be the place to mention it, but what activity events do we consider interesting?

Comment (ifireball) [reply README.md:70]: Out of scope for this PR.

**Review by Omeramsc — APPROVED** [review]
LGTM

**Pull request merged**